test/integration/connect/envoy/down.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Shorthand for when you need to clean up after a Ctrl-C or other sort of crash when run manually.